app-info/generate-eos-extra

```diff
@@ -82,13 +82,27 @@ def main():
                 len(appstream_id.split('.')) > 3):
             appstream_id = appstream_id[:-len(".desktop")]
 
+        # Add XML to include the app in the carousel on the overview page, the
+        # “Editor’s Choice” section on the overview page, the carousel on
+        # category pages, and the “Editor’s Choice” section on category pages.
+        #
+        # This will only work for apps which have a hi-res icon (defined in the
+        # main metainfo entry for that app).
```

Comment:
Interesting, does this apply only to the carousel, or anywhere?

I ask because the SOMA apps only have 64×64px icons and this is unlikely to change.

Reply:
It applies to the carousel and everywhere on the category pages. i.e. It doesn’t apply to editor’s choice on the overview page.

tl;dr: SOMA apps could be put in editor’s choice on the overview page, but not in an individual category. That’s probably OK since if you go into the Education category the SOMA apps are fairly obvious anyway (as they are quite numerous).